.github/workflows/main.yml

```diff
@@ -51,3 +51,19 @@ jobs:
       run: |
         bazel build //typescript/...
         bazel test //typescript/...
+
+    - name: Building //csharp directory (Linux)
```

Comment:
Out of curiosity, why do we have separate steps for each directory instead of one single `bazel test //...` command?

Reply:
I want to run with some flags for linux and other for mac.

Reply:
I think a better way to do this would be to add those flags to `.bazelrc`. At the top you will need this line. It will be as you pass `--config=macos` or `--config=linux` when running from one of those platforms.

```
common --enable_platform_specific_config
```

And then:

```
build:macos --host_platform @io_bazel_rules_dotnet//dotnet/toolchain:darwin_amd64_3.1.407
build:macos --platforms @io_bazel_rules_dotnet//dotnet/toolchain:darwin_amd64_3.1.407
build:linux --host_platform @io_bazel_rules_dotnet//dotnet/toolchain:linux_amd64_3.1.407
build:linux --platforms @io_bazel_rules_dotnet//dotnet/toolchain:linux_amd64_3.1.407
```

This will probably not work for remote execution yet, but perhaps better to try that later. We will need to edit `//remote_config/config:platform` to include a dotnet toolchain version constraint and possibly add more platforms.

Reply:
Done.